.mvn/gradle-enterprise.xml

```diff
@@ -4,4 +4,19 @@
     <server>
         <url>https://ge.micronaut.io</url>
     </server>
+    <buildScan>
+        <captureGoalInputFiles>true</captureGoalInputFiles>
+        <backgroundBuildScanUpload>#{isFalse(env['CI'])}</backgroundBuildScanUpload>
+        <publish>ALWAYS</publish>
```

Comment:
In the `micronaut-build` plugins we use this, but only in combination with `publishIfAuthenticated`. Is it possible to do the same here?

Reply:
Sure, I'll update it 👍